dask_geopandas/core.py

```diff
@@ -306,6 +307,34 @@ def explode(self):
     def cx(self):
         return _CoordinateIndexer(self)
 
+    def hilbert_distance(self, p=15):
+
+        """
+        A function that calculates hilbert distance for each geometry
+        in each partition of a Dask-GeoDataFrame
+
+        Parameters
+        ----------
+        p : Hilbert curve parameter
```

Comment:
It would be good to add some explanation of what this is, how it influences the result, and some guidance on whether you should set this or not